backend/compact-connect/stacks/api_stack/v1_api/staff_users.py

```diff
@@ -0,0 +1,662 @@
+from __future__ import annotations
+
+from functools import cached_property
+from typing import List
+
+from aws_cdk.aws_apigateway import Resource, MethodResponse, JsonSchema, JsonSchemaType, AuthorizationType, \
+    LambdaIntegration
+from aws_cdk.aws_cognito import IUserPool
+from aws_cdk.aws_dynamodb import ITable
+from aws_cdk.aws_kms import IKey
+from cdk_nag import NagSuppressions
+
+from common_constructs.python_function import PythonFunction
+from common_constructs.stack import Stack
+
+# Importing module level to allow lazy loading for typing
+from stacks import persistent_stack as ps
+from .. import cc_api
+
+
+class StaffUsers:
+    def __init__(
+            self, *,
+            admin_resource: Resource,
+            self_resource: Resource,
+            admin_scopes: List[str],
+            persistent_stack: ps.PersistentStack
+    ):
+        super().__init__()
+
+        self.stack = Stack.of(admin_resource)
+        self.admin_resource = admin_resource
+        self.api: cc_api.CCApi = admin_resource.api
+
+        self.log_groups = []
+        env_vars = {
+            **self.stack.common_env_vars,
+            'USER_POOL_ID': persistent_stack.staff_users.user_pool_id,
+            'USERS_TABLE_NAME': persistent_stack.staff_users.user_table.table_name,
+            'FAM_GIV_INDEX_NAME': persistent_stack.staff_users.user_table.family_given_index_name
+        }
+
+        # .../
```

Comment:
What's this comment meant to imply?

Reply:
This and the next couple of comments are just noting which url path segment the methods are creating. So this section is creating `POST` and `GET` methods right at the level of the API resource that was passed into the constructor.

Reply:
got it. thanks for changing to base-url. that makes sense!